procrastinate/sql/queries.sql

```diff
@@ -149,3 +149,35 @@ SELECT task_name AS name,
   FROM jobs AS j
  GROUP BY name
  ORDER BY name;
+
+-- list_locks --
+-- Get list of locks and number of jobs per lock
+WITH jobs AS (
+   SELECT id,
+          queue_name,
+          task_name,
+          lock,
+          args,
+          status,
+          scheduled_at,
+          attempts
+     FROM procrastinate_jobs
+    WHERE (%(queue_name)s IS NULL OR queue_name = %(queue_name)s)
+      AND (%(task_name)s IS NULL OR task_name = %(task_name)s)
+      AND (%(status)s IS NULL OR status = %(status)s)
+      AND (%(lock)s IS NULL OR lock = %(lock)s)
+)
+SELECT lock AS name,
```

Comment:
Super minor but I'd either remove the alias (`AS name`) or use `lock_name` as the alias (`AS lock_name`).